README.md.hb

```diff
@@ -496,7 +496,7 @@ methods to allow you to quickly hook-in simple inline middlewares.
 
 ### More details on creating middlewares
 
-Check the [code for existing middlewares](/src/middlewares) to see more examples
+Check the [code for existing middlewares](https://github.com/middyjs/middy/blob/master/docs/middlewares.md) to see more examples
```

Comment:
`/docs/middlewares` should route to the same place.
While still giving markdown some flexibility on choosing how to construct the url on GithHub vs the docs site.

Reply:
@ChristianMurphy Thanks for your feedback! I am not familiar with GH pages but sounds like a better approach. I will update the PR. Don't I need the `.md` ext at the end like `(/src/middlewares.md)`?

Reply:
Including the file type should be fine, and appears to be used commonly in the GitHub docs.
https://blog.github.com/2016-12-05-relative-links-for-github-pages
Keep it. :+1:

Reply:
I landed on the same page but it's not clear if GitHub pages will fallback on `.md`. Anyway, I've updated my PR. :)